xdsl/dialects/arith.py

```diff
@@ -73,14 +74,21 @@ def from_attr(attr: Attribute, typ: Attribute) -> Constant:
         return Constant.create(result_types=[typ], attributes={"value": attr})
 
     @staticmethod
-    def from_int_constant(val: Union[int, Attribute],
-                          typ: Union[int, Attribute]) -> Constant:
+    def from_int(val: Union[int, Attribute],
```

Comment:
Maybe a name that describes both arguments instead of just the first makes more sense. I guess that would become `from_int_and_width` then?

Reply:
Yes, I guess